### ISSUE TITLE
Issues while trying to connect

### ISSUE THREAD

**Issue by kakelanders** [maca134/MBCon#2]
Heya Maca,

I Believe ive set up everything properly.
Put in the proper IP and port, and directorys in config.ini aswell as done the proper configurations in my desired plugins - Restartmessages and ScheduledTasks

Yet when launching the .exe i get the following while using local IP and both 2302 (port i use) and 2303
http://pastebin.com/ydRBAz9x

Im obviously doing somthing wrong, maybe you can point me in the right direction ?

(Using this for my Arma3 Epochserver if that makes any difference)

**Comment by maca134** [comment]
in the config.ini did you set the plugin dir?

**Comment by kakelanders** [comment]
No, i did't set a patch since the plugin folder is in the approot, or do i need to pull each and every plugin i intend to use into the approot ?

Also in SchedulesTasks would this be the proper setup for a two hour restart ?

----Schedule.json----
[
    {
        "start": "120",
        "loop": -1,
        "cmd": "#restart"
    }
]

Thanks for the help Maca! :)

**Comment by Gr8z** [comment]
Looking at the logs, looks like your dlls are blocked by windows. Right Click on them > Properties > Unblock

**Comment by cikez** [comment]
Any luck getting the #restart and/or #shutdown command working in a scheduled task?
